### ISSUE TITLE
WGAN-GP for Training MSG_ProGAN is failed

### ISSUE THREAD

**Issue by Johnson-yue** [akanimax/BMSG-GAN#34]
Hi, I read your newest version paper (v3) in arxiv, and i saw that add MSG_styleGAN information, I'm glad to see it , but I have some question about paper : 

1. I think you paper said the result of showing in paper is trained MSG_ProGAN by using WGAN-GP, and trained MSG_styleGAN by using Non-saturating GAN loss with 1-sided gp.  This repo is for training MSG_ProGAN but default GAN loss is **relativistic-hinge** and then I changed it to wgan-gp it failed ,because of not support for mult-scale Discriminator , Can you update this function code. (B.T.W) I implemented by myself but I don't right or false)

2. why default using  **relativistic-hinge** not **WGAN-gp**

**Comment by akanimax** [comment]
Oh yeah indeed, thanks for pointing this out. I'll update the default loss function to WGAN-GP here too. It's possible that the loss implementation is broken now due to PyTorch upgrades so, it will need fixing. Since you have already implemented a fix, could you perhaps open a PR here?

**Comment by akanimax** [comment]
Oops, saw #35 just now! Thanks :smile:.
Will take a look...

Cheers :beers:!

**Comment by cellis** [comment]
hello, any updates on this?
